src/Gzero/Repository/ContentRepository.php

```diff
@@ -154,31 +154,21 @@ function ($query) { // default order by
     |--------------------------------------------------------------------------
     */
 
-    ///**
-    // * Get all ancestors nodes to specific node
-    // *
-    // * @param TreeNode $node    Node to find ancestors
-    // * @param int      $hydrate Doctrine2 hydrate mode. Default - Query::HYDRATE_ARRAY
-    // *
-    // * @return mixed
-    // */
-    //public function getAncestors(TreeNode $node, $hydrate = Query::HYDRATE_ARRAY)
-    //{
-    //    // root does not have ancestors
-    //    if ($node->getPath() != '/') {
-    //        $ancestorsIds = $node->getAncestorsIds();
-    //        $qb           = $this->newQB()
-    //            ->select('n')
-    //            ->from($this->getClassName(), 'n')
-    //            ->where('n.id IN(:ids)')
-    //            ->setParameter('ids', $ancestorsIds)
-    //            ->orderBy('n.level');
-    //
-    //        $nodes = $qb->getQuery()->getResult($hydrate);
-    //        return $nodes;
-    //    }
-    //    return [];
-    //}
+    /**
+     * Get all node ancestors
+     *
+     * @param Tree $node Tree node
+     *
+     * @return array|EloquentCollection|static[]
+     */
+    public function getAncestors(Tree $node)
+    {
+        // root does not have ancestors
+        if ($node->path != '/') {
+            return $node->findAncestors()->get();
```

Comment:
@MateuszUrbanowicz  I think that we should use some logic from other functions, like using eager loading to load translations etc.